apps/devhub/tests/test_views.py

```diff
@@ -1142,11 +1144,11 @@ def test_step1_submit(self):
     def test_read_dev_agreement_set(self):
         """Store current date when the user agrees with the user agreement."""
         response = self.client.get(reverse('devhub.submit.1'))
-        previous = response.context['user'].read_dev_agreement
+        assert response.context is None
 
         response = self.client.post(reverse('devhub.submit.1'), follow=True)
         user = response.context['user']
-        assert user.read_dev_agreement != previous
+        assert user.read_dev_agreement is not None
```

Comment:
use [self.assertCloseToNow(user.read_dev_agreement)](https://github.com/Nolski/olympia/blob/signing-api-agreement/apps/amo/tests/__init__.py#L402) to make sure that it has been set to `now()`